doc/source/contribution/documentation/i18n.rst

```diff
@@ -66,7 +66,7 @@ Then you need to configure your cloned repository::
 
   % cd groonga
   % ./autogen.sh
-  % ./configure
+  % ./configure --enable-document
```

Comment:
Sphinxの話はここに書いてありました。
http://groonga.org/ja/docs/contribution/documentation/introduction.html
いきなりi18nのページから見ても大丈夫なように、i18nからもリンクしておくと 
良さそうです。

Hiroshi Hatake wrote:
> In doc/source/contribution/documentation/i18n.rst
> <https://github.com/groonga/groonga/pull/501#discussion_r57097205>:
>
>> @@ -66,7 +66,7 @@ Then you need to configure your cloned repository::
>>
>>    % cd groonga
>>    % ./autogen.sh
>> -  % ./configure
>> +  % ./configure --enable-document
>
> ビルド手順に新たにsphinxのセットアップ方法を追記しないとドキュメントの通
> りにビルド出来なくなってしまうのでこれだけいきなり入れられても困ると思い
> ました。
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/groonga/groonga/pull/501/files/f1756f34340f4b1304f20bb0c275fc5591642de6#r57097205>
>

-- 
下田 洋志 <SHIMODA Hiroshi>
E-mail: shimoda@clear-code.com

株式会社クリアコード
〒170-0005 東京都豊島区南大塚3-29-9
            中野ビル3階
TEL : 03-5927-9440
FAX : 03-5927-9441
WWW : http://www.clear-code.com/